docs/community/events.md

```diff
@@ -0,0 +1,75 @@
+# Events featuring Carbyne Stack
+
+## Upcoming Events
+
+### [TPMPC '22][tpmpc22]
+
+<span class="event-title">Open Source Cloud-Native MPC (Invited Talk)</span>
+
+![TPMPC Logo](../images/events/tpmpc22.png){ class="event-logo" }
+
+JUNE 7 - 10, 2022 | AARHUS (DENMARK).
+
+!!! abstract
+
+    Not available yet.
+
+---
+
+### [StackConf '22][stackconf22]
+
+<span class="event-title">Scaling the Grail - Cloud-Native Computing on
+Encrypted Data using Carbyne Stack (Contributed Talk)</span>
+
+![Stackconf Logo](../images/events/stackconf22.png){ class="event-logo" }
+
+JULY 19 – 20, 2022 | BERLIN (GERMANY)
+
+!!! abstract
```

Comment:
how about adding a direct link to the [talk details](https://stackconf.eu/talks/scaling-the-grail-cloud-native-computing-on-encrypted-data-using-carbyne-stack/) so that interested visitors can access it immediately and, e.g., to add the calendar entry.

Reply:
Good proposal. Add the link. In addition changed the font weight for the event titles to make them more "visible".